tests/unit/expire.tcl

```diff
@@ -209,19 +209,55 @@ start_server {tags {"expire"}} {
         set e
     } {*not an integer*}
 
-    test {SET - use EX/PX option, TTL should not be reseted after loadaof} {
+    test {EXPIRE and SET EX/PX option, TTL should not be reset after loadaof} {
+        # This test makes sure that expire times are propagated as absolute
+        # times to the AOF file and not as relative time, so that when the AOF
+        # is reloaded the TTLs are not being shifted forward to the future.
+        # We want the time to logically pass when the server is restarted!
+
         r config set appendonly yes
-        r set foo bar EX 100
+        r set foo1 bar EX 100
+        r set foo2 bar PX 100000
+        r set foo3 bar
+        r set foo4 bar
+        r expire foo3 100
+        r pexpire foo4 100000
         after 2000
         r debug loadaof
-        set ttl [r ttl foo]
-        assert {$ttl <= 98 && $ttl > 90}
+        set ttl
+        assert_range [r ttl foo1] 90 98
+        assert_range [r ttl foo2] 90 98
+        assert_range [r ttl foo3] 90 98
+        assert_range [r ttl foo4] 90 98
```

Comment:
Maybe skip this when `no_latency` is used, to avoid false positives on really slow emulated envs?

Reply:
i don't think this directly matches the no_latency flag.
so far the tests it skips are ones that measure latency inside redis.
this is a time sensitive test, and we have tons of them.
since it didn't bother us till now, and i didn't change it in that respect, i tend to keep it like it is.
the alternative is to change the threshold from 90 to 80, or add a new `--no-timing-errors` flag

Reply:
We do see issues on slow systems, but I can't predict if this is going to be worse than others... let's just try and see.